src/rust/ide/src/view/node_editor.rs

```diff
@@ -459,14 +468,13 @@ impl GraphEditorIntegratedWithControllerModel {
 
         // TODO [mwu]
         //   Currently it is not possible to:
-        //    * use other project name than the default;
         //    * enter other module than the initial (namely, "Main")
         //    * describe that visualization's expression wishes to be evaluated in any other
         //      context.
         //   Because of that for now we will just hardcode the `visualization_module` using
         //   fixed defaults. In future this will be changed, then the editor will also get access
         //   to the customised values.
-        let project_name         = crate::DEFAULT_PROJECT_NAME;
+        let project_name         = self.project.project_name.as_ref();
```

Comment:
this shows wrong naming here as well. If you have a field `project`, then it should not have a field `project_name` because it already is a project, so it should not have another names. It should be `self.project.name` instead.